src/lib/stack-client.js

```diff
@@ -1,11 +1,48 @@
 import internal from 'lib/stack-internal.js'
 
+import {
+  UnavailableStackException,
+  UnauthorizedStackException
+} from './exceptions'
+
 /**
  * Cozy client instance
  * @private
  */
 let cozyClient
 
+/**
+ * Get the stackClient from the cozy-client instance
+ *
+ * @private
+ * @function
+ * @returns {Object} cozy-stack-client instance
+ */
+const getStackClient = function() {
+  return cozyClient.getStackClient()
+}
+
+/**
+ * Logout and disconnect the user
+ * @function
+ * @returns {Promise}
+ */
+const logout = function() {
+  return getStackClient().fetch('DELETE', '/auth/login').then(
```

Comment:
```
const logout = async () => {
  const resp = await getStackClient().fetch('DELETE', '/auth/login')
  ....
}
```

lets you get rid of the callback etc...

It will also be more secure in the sense that if the getStackClient() fails synchronously, it will still return a Promise that will reject instead of throwing a synchronous exception.